rules/internal/testutils/parse.go

```diff
@@ -16,14 +16,25 @@ package testutils
 
 import (
 	"bytes"
+	"compress/gzip"
 	"fmt"
 	"html/template"
+	"io/ioutil"
 	"strings"
 	"testing"
 
+	"github.com/golang/protobuf/proto"
+	dpb "github.com/golang/protobuf/protoc-gen-go/descriptor"
 	"github.com/jhump/protoreflect/desc"
 	"github.com/jhump/protoreflect/desc/protoparse"
+	"github.com/jhump/protoreflect/desc/protoprint"
 	"github.com/lithammer/dedent"
+
+	// These imports cause the common protos to be registered with
+	// the protocol buffer registry, and therefore make the call to
+	// `proto.FileDescriptor` work for the imported files.
+	_ "google.golang.org/genproto/googleapis/api/annotations"
```

Comment:
Group these common protos into a separate file, e.g., `common_imported_protos.go`, like [this one](https://github.com/googleapis/api-linter/blob/210f02b24ebe301d581cd8560e1010d909911d03/parser/internal/protos/api_common_protos.go).

Reply:
Why? It seems like it is better to have the import in the file that requires it.

Reply:
Just nitpicking.